src/iotjs_util.c

```diff
@@ -43,28 +43,29 @@ iotjs_string_t iotjs_file_read(const char* path) {
     ptr = buffer + read;
   }
 #else
-  size_t read = fread(buffer, 1, len, file);
+  size_t read = fread(buffer, 1, (size_t)len, file);
 #endif
   IOTJS_ASSERT(read == (size_t)len);
 
   *(buffer + len) = 0;
 
   fclose(file);
 
-  iotjs_string_t contents = iotjs_string_create_with_buffer(buffer, len);
+  iotjs_string_t contents =
+      iotjs_string_create_with_buffer(buffer, (uint32_t)len);
```

Comment:
As you told, if the value was returned from some int returning function, then it is okay to leave it and cast it when it is needed. But here the thing i worried was that can it make different result because of the difference between `size_t` and `uint32_t` for example on the 64 bit platform?